test/Havven.js

```diff
@@ -1760,9 +1913,13 @@ contract('Havven', async function(accounts) {
 		const debtBalance3 = await havven.debtBalanceOf(account3, nUSD);
 		await havven.burnNomins(nUSD, debtBalance3, { from: account3 });
 
-		assert.equal(debtBalance1, web3.utils.toBN('0'));
-		assert.equal(debtBalance2, web3.utils.toBN('0'));
-		assert.equal(debtBalance3, web3.utils.toBN('0'));
+		const debtBalance1After = await havven.debtBalanceOf(account1, nUSD);
+		const debtBalance2After = await havven.debtBalanceOf(account2, nUSD);
+		const debtBalance3After = await havven.debtBalanceOf(account3, nUSD);
+
+		assert.bnEqual(debtBalance1After, web3.utils.toBN('0'));
```

Comment:
`assert.bnEqual` doesn't actually require BN instances, so you could write this like this:

```suggestion
		assert.bnEqual(debtBalance1After, '0');
```